packages/graphql-mesh-server/lib/fargate.ts

```diff
@@ -216,5 +233,57 @@ export class MeshService extends Construct {
             ],
             adjustmentType: auto_scaling.AdjustmentType.CHANGE_IN_CAPACITY,
         });
+
+        // Cross account role to get read only access to mesh
+        // and the relevant logs
+        if (props.readOnlyRoleName) {
```

Comment:
This should probably be `(props.readOnlyRoleName && props.awsAccountArn)` OR just check `props.awsAccountArn` and set a default for `props.readOnlyRoleName `

Reply:
@AdamJHall done! Ready for another round